e2e/jest.test-setup.ts

```diff
@@ -31,10 +30,11 @@ beforeEach(async () => {
       const body = requestBody.length === 0 ? '' : `\n${requestBody}`;
       logger.log('info', 'Request issued: %s %s %s', request.method(), request.url(), body);
     }
-    try {
-      request.continue();
-      // tslint:disable-next-line:no-empty
-    } catch (e) {}
+    return Promise.resolve()
+      .then(() => request.continue())
+      .catch(() => {
+        // Ignored
+      });
```

Comment:
A workaround to resolve Puppeteer tool failure.

Test [failure](https://app.circleci.com/pipelines/github/all-of-us/workbench/9314/workflows/8c3cff57-ddb4-43f0-811c-69600ff311b0/jobs/133165) stacktrace:
```
Error: Request is already handled!
    at Object.exports.assert (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/assert.js:26:15)
    at HTTPRequest.continue (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/HTTPRequest.js:217:21)
    at /home/circleci/workbench/e2e/jest.test-setup.ts:63:27
    at /home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/vendor/mitt/src/index.js:51:62
    at Array.map (<anonymous>)
    at Object.emit (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/vendor/mitt/src/index.js:51:43)
    at Page.emit (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/EventEmitter.js:72:22)
    at /home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:146:108
    at /home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/vendor/mitt/src/index.js:51:62
    at Array.map (<anonymous>)
    ```